### ISSUE TITLE
Request - Add day of week and day of month

### ISSUE THREAD

**Issue by SoBaldrick** [Qrome/marquee-scroller#282]
I love the clock, but would be great if on the 8 wide version, it could show time, day of week and day on month as below all the time, would help with my Moms Dementia.

2: 49 . Mon 06

3:17. Tue 31
 
HH:MM. Www DD

HH - hour

MM - min

Www - day of week

DD - day of month

anyone able to help

**Comment by Qrome** [comment]
I like this.   If I get some time I will take a look at it.  Should not be too hard to add another display option for the 8 wide.

**Comment by SoBaldrick** [comment]
Thank you, would be a much appreciated

**Comment by rob040** [comment]
I was also looking at this, but it won't fit.
"HH:MM Www DD" are 12 characters. Each character has 6x8 pixels, echa tile is 8x8 pixels, 12x6/8=9. It needs 9 tiles. 
For an 8 tile display, need to lose 2 characters. We would get a bit of an awkward format like 
"HH:MM WwDD".
Would that work?

**Comment by SoBaldrick** [comment]
Yes, that would work fine, or just go for HH:MM Www which may be better

**Comment by rob040** [comment]
I have looked at it and decided to implement a format like "HH:MMWwwDD"

On screen it looked quite readable, due to spacing in the characters itself.

Have a look at my fork https://github.com/rob040/LEDmatrixClock/releases, and let me know how what you think of it.

**Comment by SoBaldrick** [comment]
Great, thank you, I’ll have a look and let you knowAndrewSent from my iPadOn 5 Jul 2025, at 15:41, rob040 ***@***.***> wrote:﻿rob040 left a comment (Qrome/marquee-scroller#282)
I have looked at it and decided to implement a format like "HH:MMWwwDD"
On screen it looked quite readable, due to spacing in the characters itself.
Have a look at my fork https://github.com/rob040/LEDmatrixClock/releases, and let me know how what you think of it.
